test/unit/util/operator/test-operators-basic.cpp

```diff
@@ -0,0 +1,102 @@
+//~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~//
```

Comment:
Would it be clearer to replace "basic" with "math" in the filename and test names?

Reply:
Sure, will do